src/helpers/period.coffee

```diff
@@ -0,0 +1,7 @@
+_ = require 'underscore'
+
+PeriodHelper =
+  getOrder: (period) ->
+    parseInt(period.name)
```

Comment:
Will `parseInt` work for non-integer periods like 'Block B'? I assume it will return `NaN` and not sort at all